sender.go

```diff
@@ -48,6 +48,8 @@ import (
 // chars found in table or column name.
 var ErrInvalidMsg = errors.New("invalid message")
 
+const fsFileNameLimit = 127
```

Comment:
this limit is configurable on server-side isn't it?

Reply:
it is, this is also configurable limit on the server. Can this perhaps be made configurable on client too ?

Reply:
My assumption was that 99.(9)% of users won't change the server-side default. But let me make it configurable since it won't hurt.